### PR TITLE
902

### PR DESCRIPTION
**Pull request by jospoortvliet**
@cmonteroluque OK or changes since this morning? Do you want to leave out the bookmarks app (I think it's fine to do that next round)?

* [x] changelogs
* [x] updated strings for tarballs
* [ ] updated strings for VM's ( @jnweiger could you do a PR for that once this is merged?)
* [x] updated release channels page

### CODE REVIEW

File: page-changelog.php [hunk 1]
```diff
@@ -12,8 +12,217 @@
 <li><a href="#latest6">ownCloud 6 (unsupported!)</a></li>
 </ul>
 
-<a name="latest9"></a>
```
Comment (RealRancor) [inline page-changelog.php]: Accidentally deleted?

File: page-changelog.php [hunk 2]
```diff
@@ -12,8 +12,217 @@
...
+<li>[Stable 9] Make ownCloud work again in php 7.0.6 - <a href="https://github.com/owncloud/core/issues/24343">#24343</a></li>
+<li>After switching to Postgresql changing any activity checkbox checks all of the boxes - <a href="https://github.com/owncloud/core/issues/23761">#23761</a></li>
+<li>Upgrade from o.C. 8.2.X to 9.0.X. Cannot access to oC if and LDAP user that made a share was deleted from the LDAP server. - <a href="https://github.com/owncloud/core/issues/24090">#24090</a></li>
+<li>Update sabre/dav to 3.0.9 for windows 10 support - <a href="https://github.com/owncloud/3rdparty/issues/269">3rdparty/#269</a></li>
+<li>Update sabre/dav to 3.0.9 on stable9 - <a href="https://github.com/owncloud/3rdparty/issues/270">3rdparty/#270</a></li>
+<li>Spinning wheel OC 9 - <a href="https://github.com/owncloud/activity/issues/481">activity/#481</a></li>
+<li>wrong links in RSS feed - <a href="https://github.com/owncloud/activity/issues/508">activity/#508</a></li>
+<li>LDAP users don't get listed when I try to share - <a href="https://github.com/owncloud/android/issues/1527">android/#1527</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: Changes of the android app in the core changelog?

File: page-changelog.php [hunk 3]
```diff
@@ -12,8 +12,217 @@
...
+<li>Export Addressbook - <a href="https://github.com/owncloud/contacts/issues/248">contacts/#248</a></li>
+<li>Review dav urls as used in the docs - <a href="https://github.com/owncloud/documentation/issues/2074">documentation/#2074</a></li>
+<li>Acessing files via webdav with curl: wrong link - <a href="https://github.com/owncloud/documentation/issues/2330">documentation/#2330</a></li>
+<li>Remove non-existent warning for file locking - <a href="https://github.com/owncloud/documentation/issues/2351">documentation/#2351</a></li>
+<li>update php5-smbclient status - <a href="https://github.com/owncloud/documentation/issues/2362">documentation/#2362</a></li>
+<li>Re-added some performance hints. - <a href="https://github.com/owncloud/documentation/issues/2367">documentation/#2367</a></li>
+<li>Performance tuning: Recommend redis based file locking - <a href="https://github.com/owncloud/documentation/issues/2381">documentation/#2381</a></li>
+<li>Ubuntu 16.04 - <a href="https://github.com/owncloud/documentation/issues/2396">documentation/#2396</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: Changes of the documentation in the core changelog?

File: page-changelog.php [hunk 4]
```diff
@@ -12,8 +12,217 @@
...
+<li>[stable9] Redirect page if not shown in an iframe - <a href="https://github.com/owncloud/files_pdfviewer/issues/109">files_pdfviewer/#109</a></li>
+<li>[Stable9] update version to what is in core files external - <a href="https://github.com/owncloud/objectstore/issues/40">objectstore/#40</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: Not accessible by CE users

File: page-changelog.php [hunk 5]
```diff
@@ -12,8 +12,217 @@
...
+<li>integrity-check command shows error after running updater from 9.0.1 stable to daily stable9 - <a href="https://github.com/owncloud/updater/issues/342">updater/#342</a></li>
+<li>updater/update.log has to be stored in the data folder - <a href="https://github.com/owncloud/updater/issues/317">updater/#317</a></li>
+<li>3rdparty folder not overwritten on update - <a href="https://github.com/owncloud/updater/issues/316">updater/#316</a></li>
+<li>getExtractionBaseDir should be deleted before code signature is verified - <a href="https://github.com/owncloud/updater/issues/318">updater/#318</a></li>
+<li>Upgrade from 9.0 to 9.1 fails - <a href="https://github.com/owncloud/updater/issues/336">updater/#336</a></li>
+<li>Wrong app path when optional PCNTL module was not enabled - <a href="https://github.com/owncloud/updater/issues/335">updater/#335</a></li>
+<li>Option to delete empty folders after x days - <a href="https://github.com/owncloud/workflow/issues/83">workflow/#83</a></li>
+<li>[9.0] Parent tags missing in userretention - <a href="https://github.com/owncloud/workflow/issues/87">workflow/#87</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: Not accessible by CE users

**Comment by RealRancor** [comment]
There now are also two latest8.0 and two latest7 anchors in the changelog

File: page-changelog.php (hunk 2 above)
Comment (jospoortvliet) [inline page-changelog.php]: yeah, was a core related issue. But took it out anyway - it was raised in core.

File: page-changelog.php (hunk 3 above)
Comment (jospoortvliet) [inline page-changelog.php]: I took it out. @cmonteroluque can you filter this next time? I noticed but thought it was intentional...

File: page-changelog.php (hunk 5 above)
Comment (jospoortvliet) [inline page-changelog.php]: good catch.

**Comment by jospoortvliet** [comment]
Thanks a billion for the review, @RealRancor went over the changelog again to remove the rest of the enterprisy stuff. Clearly a search for 'enterprise' doesn't do it ;-)

**Comment by cmonteroluque** [comment]
@jnweiger ping

**Pull request merged**